dpgen2/entrypoint/submit.py

```diff
@@ -206,7 +206,7 @@ def make_naive_exploration_scheduler(
     sys_configs = (
         config["sys_configs"] if old_style else config["explore"]["configurations"]
     )
-    sys_prefix = config.get("sys_prefix")
+    sys_prefix = config["explore"].get("configuration_prefix")
```

Comment:
`configuration_prefix` is removed in pr #175, because it is never used. 
The configuration from file is supported by the [`file` type](https://docs.deepmodeling.com/projects/dpgen2/en/latest/submit_args.html#explore-lmp-configurations-file) of configuration generator.  
you man find an example in https://github.com/wanghan-iapcm/dpgen2-1/blob/improve-doc/docs/input.md#exploration

Reply:
we will remove `configuration_prefix` in a new PR